application/core/services/export_service.py

```
@@ -0,0 +1,178 @@
+            else:
+                raise ValueError(f"不支持的导出格式: {format}")
+            
+            logger.info(f"导出成功，文件大小: {len(result[0])} 字节")
+            return result
+        except Exception as e:
+            logger.error(f"导出章节失败: {str(e)}")
+            raise
+    
+    def _export_to_epub(self, novel: Novel, chapters: list[Chapter]) -> Tuple[bytes, str, str]:
+        """导出为EPUB格式"""
+        # 这里使用临时实现，实际需要使用专业的EPUB生成库
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)